dali/python/nvidia/dali/ops/__init__.py

```diff
@@ -70,7 +84,7 @@ def _separate_kwargs(kwargs, arg_input_type=_DataNode):
     def is_arg_input_type(x):
         return isinstance(x, arg_input_type)
 
-    def is_call_arg(name, value):
+    def is_arg_input(name, value):
```

Comment:
This is not a correct name. The argument "name" is passed to `__call__` even though it's not an argument input. The question is whether we can somehow rectify this and pass `name` to the `__init__` instead.

Reply:
Done